src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -150,11 +148,18 @@ export default function Overlay() {
         style.backgroundImage !== "none" ||
         style.backgroundColor !== "rgba(0, 0, 0, 0)"
       ) {
-        return;
+        return true;
```

Comment:
Just to improve the utility here, I'd return the element here, and otherwise return nothing. Your if statements below won't change, but just makes this method slightly more flexible in the future.